terms/lossfunctions_ag.tex

```diff
@@ -0,0 +1,87 @@
+\chapter{Loss Functions}
+\label{ch:loss functions}\index{loss functions}
+
+The driving force of any mathematical optimization problem, especially in machine learning, is the loss function, measuring how well an algorithm is performing with regards to its target. In general terms, the evaluation for any learning problem is done by taking a predicted or estimated output and compare it to the relative real recorded outcome, typically in a dataset. The expected value of the loss is called the statistical risk. The formulation of the loss function is crucial and must be accurately defined according to the selected algorithm and the selected data for the particular optimization problem. The loss function must reproduce a numerical value that somehow reflects the level of achievement desired by the algorithm.  In machine learning we are typically attempting to find the parameters that fit our model and that minimizes the loss function. Minimizing the loss function is one of the fundamental foundations of statistical machine learning \citep{pmlr-v32-yanga14}. 
+
+\begin{marginfigure}%
+	\includegraphics[width=\linewidth]{loss_functions/cross_entropy.png}
+	\caption{Log Loss Function}
+	\label{fig:logloss}
+\end{marginfigure}
+
+
+The three canonical loss functions as described by Vapnik \citep{vapnik1998statistical} are:
+\begin{itemize} \item The 0-1 Loss – the simplest loss function mainly used in binary classification \begin{equation} L(y,\hat{y)} \end{equation}, resulting in an error of 0 if and only if $y == \hat{y}$. Otherwise the function will produce the value of 1. This does not however differentiate between different classes and types of errors. Variations of this function may thus be formulated by extending the simple loss function and make the resulting loss dependent on another function.
```

Comment:
This is still broken `L(y,\hat{y)}` and there are many such occurances.  The second bracket should be outside the hat, so `L(y,\hat{y})`...